.github/workflows/tests.yml

```
@@ -96,6 +96,36 @@ jobs:
       - name: Run test suite on local python version
         run: tox -e ${{ steps.set_tox_env.outputs.tox_env }} --current-env
 
+  test-coverage:
+    name: "Check the test coverage"
+    runs-on: ubuntu-latest
+    needs:
+      - pre_job
+    if: needs.pre_job.outputs.should_skip == '{}' || !fromJSON(needs.pre_job.outputs.paths_result).main.should_skip
```

```suggestion
    if: needs.pre_job.outputs.should_skip == '{}' || !fromJSON(needs.pre_job.outputs.paths_result).python.should_skip
```
This `main` ref does not exist in the skip_check step's paths_filter definition.